### ISSUE TITLE
Displaying MEI files with TEI Publisher

### ISSUE THREAD

**Issue by adnilem** [DCMLab/ddd#37]
To display MEI files with TEI Publisher, I worked with TEI Publisher's "Playground" ([https://teipublisher.com/exist/apps/tei-publisher/index.html?tab=0](https://teipublisher.com/exist/apps/tei-publisher/index.html?tab=0)) online and on a local server, as well as with their [webcomponents](https://teipublisher.com/exist/apps/tei-publisher/doc/documentation.xml?odd=docbook.odd&view=div&id=embedding-examples) functionality (together with Github Pages). I also used some live examples from the [doc](https://unpkg.com/@teipublisher/pb-components@1.17.1/dist/api.html#dts-client.0). 
The goal is to have XML/TEI files with embedded MEI displayed on our website. 

Issue with MEI files:
- Putting a TEI element and then a MEI element within the same file doesn't work.
- `<pb-mei>`, part of the webcomponents, works with samples from [verovio](https://verovio.humdrum.org/) and MEI's [github](https://github.com/music-encoding/sample-encodings), but not our files. The url asked is a raw.github link, the uploaded file can be txt or mei 
<img width="400" alt="Screenshot 2021-01-27 at 19 04 50" src="https://user-images.githubusercontent.com/43856420/106033968-9db1b580-60d2-11eb-9583-91536e32c4df.png">

_The MEI player: top is a file from Verovio, bottom from our project: no content._

Issues with embedding MEI into TEI:
- Copy pasting a MEI file into a TEI element doesn't work, not matter where the `meiHead` element is. Some text from the MEI element are displayed but no music or payer. 
- Changing the ODD/Templates in the TEI Publisher app doesn't do anything. Modifying a specific ODD also has no effect. 
- There are some server erros: some files will display fine when working locally (ref is URS from TEI Publisher app where the file is uploaded) but returns a server error once pushed and accessed from the test website's url. Sometimes, it also fails to load when working locally but I couldn't find a pattern. 
It has probably something to do with the document's structure: [https://faq.teipublisher.com/general/no-content/](https://faq.teipublisher.com/general/no-content/).

**Comment by fabianmoss** [comment]
Thanks for creating the issue! I have some comments:

1. It is not necessarily the goal to **embed** the MEI code in the TEI. We just have to find a way to display rendered MEI on the webpage.
2. Since the `<pb-mei>` component works with the sample files, I suspect that the files I created are somehow broken. I will check that! Can you please confirm that you can display sample files from Verovio on our GitHub page? Or is the screenshot taken from that?

In short, we should not spend too much time trying to embed MEI in TEI directly. 
If the `<pb-mei>` component works, that's fine! 

But just to be sure, could you please also try the solution mentioned here: https://www.verovio.org/javascript.xhtml (directly using Verovio, without the TEI Publisher component)?

**Comment by adnilem** [comment]
I tried Verovio's viewer with the files from [this folder](https://github.com/adnilem/test-site/tree/main/MEI). The sample files that I got from Verovio work fine, the ones you gave me (including one that I modified to add extra metadata/tags so it would be more similar to the Verovio's files) give an error message:
<img width="643" alt="Screenshot 2021-01-29 at 16 44 21" src="https://user-images.githubusercontent.com/43856420/106296353-d88e2780-6251-11eb-921b-ea8046562dd8.png">
The problem definitely comes from the files.

The screenshot is from a post that I made on a [forked version](https://github.com/adnilem/ddd/tree/page-draft) of the project page as I wasn't sure we wanted the post online yet...
